qpc_tools/server/ansible/install/roles/server_podman/tasks/redhat_registry.yml

```diff
@@ -0,0 +1,12 @@
+---
+
+- name: Login to redhat registry with podman
+  shell: "podman login -u '{{rh_registry_username}}' -p '{{rh_registry_password}}' registry.redhat.io"
+  register: rh_registry_login_attempt
+  ignore_errors: True
+  no_log: True
+
+- fail:
+    msg: "We failed to login for you, you should log in with `podman login registry.redhat.io`"
```

Comment:
maybe we should change this message to "Login attempt failed. Log in manually with `podman login registry.redhat.io`"